### ISSUE TITLE
Inconsistent API: strings instead of numbers

### ISSUE THREAD

**Issue by tikiki** [WikiJourney/wikijourney_website#32]
The API is weird: in the POI array, it returns a number for "latitude" and "longitude", but for the "user_location", it returns strings instead.
This requires additional parsing for the client side...
```
{

    "infos":{
        "source":"WikiJourney API",
        "link":"http:\/\/wikijourney.eu\/",
        "api_version":"Beta 1.1"
    },
    "user_location":{
        "latitude":"48.8582602",                                 // There, string
        "longitude":"2.29449905431968"                           // There too
    },
    "poi":{
        "nb_poi":10,
        "poi_info":[
            {
                "latitude":48.858333,                            // These are numbers as expected
                "longitude":2.294167,
                "name":"tour Eiffel",
                "sitelink":"https:\/\/fr.wikipedia.org\/wiki\/Tour_Eiffel",
                "type_name":"tour en treillis",
                "type_id":1440476,
                "id":243,
                "image_url":"https:\/\/upload.wikimedia.org\/wikipedia\/commons\/9\/93\/Paris_arr_jms-298px.gif"
            },
...
```

**Comment by sylvainar** [comment]
Sorry, it's a mistake. I correct this as soon as possible.

**Comment by sylvainar** [comment]
Fixed in commit c455a3e4429a6e4030a26eb40be214217652bc39 in WikiJourney/wikijourney_api :smiley:
